src/screens/HomeStack/BrowseBikes.jsx

```diff
@@ -2,17 +2,12 @@ import React, { useState, useEffect } from 'react';
 import * as firebase from 'firebase';
 import PropTypes from 'prop-types';
 import { Alert } from 'react-native';
-import {
-  Container,
-  Content,
-  Grid,
-  Row,
-  Text,
-  Spinner
-} from 'native-base';
+import { Container, Content, Grid, Row, Text, Spinner } from 'native-base';
 import BrowseBikesMap from '../../components/BrowseBikesMap';
 import BrowseBikesList from '../../components/BrowseBikesList';
 import { getBikesWithinRadius } from '../../api/bikes';
+// eslint-disable-next-line no-unused-vars
+import { bikeRatings } from '../../api/bikeRatings';
```

Comment:
I don't think we need to import something if it's not being used. I'm going to merge this in and fix it in my PR though so we don't lose velocity over something that's just a small nitpick.